pkg/power/uptime.go

```diff
@@ -71,6 +72,23 @@ func (u *Uptime) uptime(ctx context.Context) error {
 		u.Mark.Signal()
 
 		log.Info().Str("hash", hash.Hex()).Msg("node uptime hash")
+		return nil
+	}
+	for {
+		unusable := false
+		if app.CheckFlag(app.ReadonlyCache) {
+			log.Error().Msg("node cache is read only")
+			unusable = true
+		}
+		if app.CheckFlag(app.LimitedCache) {
+			log.Error().Msg("node is running on limited cache")
+			unusable = true
+		}
+		if unusable {
+			log.Error().Msg("node is not usable skipping uptime reports")
+		} else if err := report(); err != nil {
+			return err
+		}
```

Comment:
why it's done like this but no like
```suggestion
		if app.CheckFlag(app.ReadonlyCache) {
			log.Error().Msg("node cache is read only")
		} else if app.CheckFlag(app.LimitedCache) {
			log.Error().Msg("node is running on limited cache")
		} else if err := report(); err != nil {
			return err
		}
```

Also I think that's the wrong place, because the SendNow function can sitll be called from other places (for exasmple node shutdown) when it actually shouldn't.

Maybe the better place to palce these checks in the SendNow fuction.

Reply:
the assumption was that both flags (and other flags added) can occur at the same time so we need to log to the farmer every problem with the node. or should we just report one?

Reply:
regardless if both can happen on the same time or not (they won't) but i don't think the logs need to be this verbose here (in other words it doesn't really matter here what is the reason we skipping the uptime) because:
- The healthcheck will still log the proper rason 
- The Ui will still show the proper reason.

This makes me think actually this can be even more comact by doing `if is limite-cache or is reasony, then skip`